index.bs

```diff
@@ -1163,6 +1163,16 @@ Meeting Scheduling and Announcements</h4>
 	Shorter notice for a meeting is allowed
 	provided that there are no objections from group participants.
 
+<h4 id="meeting-minutes">
+Meeting Minutes</h4>
+
+	Groups are expected to take and retain <dfn>minutes</dfn> of their meetings,
+	and <em class="rfc2119">must</em> record in particular
+	any official [=group decisions=] made during the meeting discussions.
+	Precise details of the discussion leading to such decisions
+	is recommended, but not required
+	provided that the rationale for the [=group decision=] is nonetheless clear.
```

Comment:
provided that the rationale, and consensus of support, for the …

Reply:
It's not a [=group decision=] if there wasn't consensus backing as determined by the chair (or a vote, or ...; there's an entire section on [decisions when there's no consensus](https://www.w3.org/Consortium/Process/Drafts/#managing-dissent)), so I would rather not include anything about consensus here. If a group just wants to take minutes as a list of 

RESOLVED: thing decided
RATIONALE: why we did it
RESOLVED: other thing decided
RATIONALE: why we did it

That should be acceptable. (Might even be more understandable than the transcription style for groups who tend to have very sparse transcribing.)

Reply:
OK. I suppose that we can leave to the group/chair the evidence that consensus was measured somehow.